libs/accounts/src/lib/accounts-manager.tsx

```diff
@@ -57,7 +58,7 @@ export const AccountManager = ({
     successCallback(rowsThisBlock, lastRow);
   };
   return (
-    data && (
+    <AsyncRenderer data={data || []} error={error} loading={loading}>
```

Comment:
this is to not show "No data" but rather display the ag-grid headers and show "No accounts"